### ISSUE TITLE
[BUG] Indexer found no files with included filetypes in current cache locations [0x14]

### ISSUE THREAD

**Issue by WittyMao** [Xanashi/Icaros#88]
Indexing Completed: Indexer found no files with included filetypes in current cache locations [0x14]
The same computer username and computer name can cause 0x14 error.

![PixPin_2023-12-30_01-13-14](https://github.com/Xanashi/Icaros/assets/24718923/5ec041b2-8630-4f46-9aa6-7fa3e12f1e81)

After changing the computer name, the program can run normally.

![PixPin_2023-12-30_01-17-16](https://github.com/Xanashi/Icaros/assets/24718923/b63089f6-2cfa-4015-ac20-7eb0cfc46ee1)

**Comment by Xanashi** [comment]
Thank you for reporting this bug. 
I'm not sure why the computer name or username would affect the cacher indexer, but I'll try to reproduce this as soon as possible.
It could definitely explain, why I still see reports of the 0x14 error without being able to reproduce them myself.

**Comment by AlanEdd** [comment]
(Edit)
I was getting this on 3.2.1 but updated to 3.2.3 and it seems to be indexing/processing.
My machine and user name are different. It just started not working for some reason. The .NET thing seems to be updated to 4.8 I think (regular updates on work machine).
![image](https://github.com/Xanashi/Icaros/assets/36997610/d7099bbe-1bca-4622-8373-5578a312578e)

*I have removed the rest of the post as it's irrelevant to this thread. I am having some issues but I suspect it's something I have done rather than this software.

**Comment by Xanashi** [comment]
Hi AlanEdd,
So you don't have the indexing issue any longer after updating to 3.3.2?
(Just so I know there isn't also another 0x14 issue besides the naming issue mentioned above)

**Comment by AlanEdd** [comment]
Yes running the latest version and it is working fine. I was having a lot of issues with OneDrive, thinking that it was Icaros not working but turns out it's a well known issue with One Drive messing up the generation of thumbnails. All generate perfectly with Icaros and One Drive is wiping them. There is no solution at the moment except to take stuff out of One Drive until Microsoft address the issue.
3.3.2 indexing fine and with paths with a special character 'é'
Thank you.

**Comment by WittyMao** [comment]
> Thank you for reporting this bug. I'm not sure why the computer name or username would affect the cacher indexer, but I'll try to reproduce this as soon as possible. It could definitely explain, why I still see reports of the 0x14 error without being able to reproduce them myself.

Thank you for the reply.
I tested issue on a newly installed Windows in a virtual machine, and I hope this can help you.

**Comment by Xanashi** [comment]
> > Thank you for reporting this bug. I'm not sure why the computer name or username would affect the cacher indexer, but I'll try to reproduce this as soon as possible. It could definitely explain, why I still see reports of the 0x14 error without being able to reproduce them myself.
> 
> Thank you for the reply. I tested issue on a newly installed Windows in a virtual machine, and I hope this can help you.

Thank you for letting me know. I'll make sure to test it and see if I can reproduce the bug.

**Comment by ASHINMAY** [comment]
I also encountered this BUG, ​​the reason is also "same computer username and computer name", the software version is 3.3.4.

**Comment by V7MoVknqxavTR** [comment]
Still having this issue, no idea what the problem is. How do I generate or see the error logs?
